### PR TITLE
Add a Target Group non-error computed metric

### LINKED ISSUE #5: Create a Target Group Computed Metric for 2xxs/(2xxs + 5xxs)
This is a specialized metric to specifically capture 2xxs vs 5xxs.

### PR DESCRIPTION
**Pull request by TheConnMan**
Closes #5

### CODE REVIEW

File: analyticConfigurations/computed_metric-aws.tag.json [hunk 1]
```diff
@@ -18,6 +18,13 @@
           "fqn": "netuitive.aws.applicationelb.httpsuccesspercent",
           "name": "HTTP Success Rate"
         }
+      }, {
+        "match": "netuitive.aws.applicationelb.httpnonerrorpercent",
+        "properties": {
+          "cml": "100 * ${aws.applicationelb.httpcode_target_2xx_count}.actual / (${aws.applicationelb.httpcode_target_2xx_count}.actual + ${aws.applicationelb.httpcode_target_5xx_count}.actual)",
+          "fqn": "netuitive.aws.applicationelb.httpnonerrorpercent",
+          "name": "HTTP Non-Error Rate"
```
Comment (Grumblesmith) [inline analyticConfigurations/computed_metric-aws.tag.json]: This name is a bit confusing to me. This metric counts the rate of successes versus server errors, leaving out redirection and client errors, right?

File: analyticConfigurations/computed_metric-aws.tag.json [hunk 2]
```diff
@@ -18,6 +18,13 @@
           "fqn": "netuitive.aws.applicationelb.httpsuccesspercent",
           "name": "HTTP Success Rate"
         }
+      }, {
+        "match": "netuitive.aws.applicationelb.httpnonerrorpercent",
+        "properties": {
+          "cml": "100 * ${aws.applicationelb.httpcode_target_2xx_count}.actual / (${aws.applicationelb.httpcode_target_2xx_count}.actual + ${aws.applicationelb.httpcode_target_5xx_count}.actual)",
```
Comment (Grumblesmith) [inline analyticConfigurations/computed_metric-aws.tag.json]: Does cml have divide by zero checking built into it, such that you don't need to check that 2xx+5xx != 0?

**Review by Grumblesmith — CHANGES_REQUESTED** [review]

File: analyticConfigurations/computed_metric-aws.tag.json (hunk 2 above)
Comment (TheConnMan) [reply analyticConfigurations/computed_metric-aws.tag.json]: It does. I can't find the PR where @jemassey told me it does.

File: analyticConfigurations/computed_metric-aws.tag.json (hunk 1 above)
Comment (TheConnMan) [reply analyticConfigurations/computed_metric-aws.tag.json]: I had a hard time coming up with a name for this. Any name recommendations are welcome.

Comment (Grumblesmith) [reply analyticConfigurations/computed_metric-aws.tag.json]: How about "HTTP Server Errors to Success Ratio"?

Comment (TheConnMan) [reply analyticConfigurations/computed_metric-aws.tag.json]: It's not 5xx/2xx, though, so it's not really a ratio.

Comment (Grumblesmith) [reply analyticConfigurations/computed_metric-aws.tag.json]: But it's also not a rate, and it's certainly not measuring HTTP Non-Errors

Comment (TheConnMan) [reply analyticConfigurations/computed_metric-aws.tag.json]: So are we coming to the conclusion that this metric has no place in this package?

Comment (Grumblesmith) [reply analyticConfigurations/computed_metric-aws.tag.json]: I'm not sure if that's the conclusion, but I'm still not exactly sure what this metric is measuring. I'm not willing to release it until we know that and can name it appropriately.

Comment (rclagett) [reply analyticConfigurations/computed_metric-aws.tag.json]: I agree with @Grumblesmith. I'd also like to know the problem it's trying to solve.

File: README.md [hunk 3]
```diff
@@ -6,6 +6,8 @@ For detailed information on this package, please refer to the [online documentat
 
 ### Version next
 
+* Add a Target Group non-error computed metric
```
Comment (Grumblesmith) [inline README.md]: @TheConnMan could you change the metric name here, too?

**Review by Grumblesmith — CHANGES_REQUESTED** [review]

**Review by Grumblesmith — APPROVED** [review]

**Review by rclagett — APPROVED** [review]

**Review by bspindler — APPROVED** [review]

**Pull request merged**